src/swarm/neo/client/mixins/ClientCore.d

```diff
@@ -857,7 +855,24 @@ template ClientCore ( )
 
     public void abortAllRequests ( )
     {
-        // TODO
+        this.connections.request_set.abortAllRequests();
```

Comment:
Bearing in mind the comment below, should this method also reset all the connections?